### PR TITLE
Remove inactive mailing lists

### LINKED ISSUE #65: Remove Summarization Logic for Inactive bitcoin-dev and lightning-dev Mailing Lists
Our summarization tool currently includes logic to check for new posts and update summaries for both the `bitcoin-dev` and `lightning-dev` mailing lists. However, both of these mailing lists are now inactive.

The `bitcoin-dev` and `lightning-dev` lists have no new posts, but our nightly cron jobs still check them and try to update summaries for older discussions. This is no longer necessary.

**Task**

Remove the logic that checks for new posts and updates summaries for the following domains:
  - `https://lists.linuxfoundation.org/pipermail/bitcoin-dev/`
  - `https://lists.linuxfoundation.org/pipermail/lightning-dev/`

### PR DESCRIPTION
**Pull request by urvishp80**
closes #65

### CODE REVIEW

File: generate_weekly_newsletter_json.py [hunk 1]
```diff
@@ -19,10 +19,8 @@
     xml_gen = GenerateXML()
     elastic_search = ElasticSearchClient()
     dev_urls = [
-        ["https://lists.linuxfoundation.org/pipermail/bitcoin-dev/",
-         "https://gnusha.org/pi/bitcoindev/",
+        ["https://gnusha.org/pi/bitcoindev/",
          "https://mailing-list.bitcoindevs.xyz/bitcoindev/"],
```
Comment (kouloumos) [inline generate_weekly_newsletter_json.py]: here we have an array inside an array?

File: generate_homepage_xml.py [hunk 2]
```diff
@@ -46,10 +46,8 @@ def page_data_handling(data_list: list, get_unique_per_dev=False):
     xml_gen = GenerateXML()
     elastic_search = ElasticSearchClient()
     dev_urls = [
-        ["https://lists.linuxfoundation.org/pipermail/bitcoin-dev/",
-         "https://gnusha.org/pi/bitcoindev/",
+        ["https://gnusha.org/pi/bitcoindev/",
          "https://mailing-list.bitcoindevs.xyz/bitcoindev/"],
```
Comment (kouloumos) [inline generate_homepage_xml.py]: here we have an array inside an array?

File: src/xml_utils.py [hunk 3]
```diff
@@ -84,12 +84,12 @@ def read_xml_file(self, full_path):
 
         link = root.findall(".//atom:entry/atom:link", namespaces)[0].get('href')
         if "bitcoin-dev" in link:
-            domain = "https://lists.linuxfoundation.org/pipermail/bitcoin-dev/"
+            domain = "https://gnusha.org/url/https://lists.linuxfoundation.org/pipermail/bitcoin-dev/"
```
Comment (kouloumos) [inline src/xml_utils.py]: our replacement script only replaces `url` and `thread_url` of documents, right? which means that this `domain` will never match.

**Review by kouloumos — COMMENTED** [review]
The code in `generate_homepage_xml.py` is also responsible for generating the "Today in History" section. Although the lists are inactive, we want those sections to be created for those lists as they refer to previous posts.

I also left a couple of comments inline.

File: generate_weekly_newsletter_json.py (hunk 1 above)
Comment (urvishp80) [reply generate_weekly_newsletter_json.py]: Yes, an array inside an array handle the unique mailing lists, e.g., bitcoin-dev have multiple domains but we want to show only one post related to in "Today in History", it doesn't affect the other logics such as Active Posts and Recent Posts.

File: generate_homepage_xml.py (hunk 2 above)
Comment (urvishp80) [reply generate_homepage_xml.py]: Same as above - https://github.com/bitcoinsearch/summarizer/pull/66#discussion_r1838331497

File: src/xml_utils.py (hunk 3 above)
Comment (urvishp80) [reply src/xml_utils.py]: Let me fix it!

**Comment by kouloumos** [comment]
reopening

**Comment by kouloumos** [comment]
@urvishp80 I mentioned the need for "Today in History" section in `generate_homepage_xml.py` but you also restored the new links in
- `generate_weekly_newsletter_json.py`
- `push_summary_to_es.py`
- `xmls_generator_production.py`

why is that?

**Comment by urvishp80** [comment]
@kouloumos We can drop these two inactive URLs from `xmls_generator_production.py` and  `push_summary_to_es.py`  file as we are sure that there won't be any new documents from the given two mailing lists in the future.

And for `generate_weekly_newsletter_json.py` and `generate_homepage_xml.py` we need to keep them as we are using it for "Ongoing Discussions" and "Today in History/Active Posts" respectively.

**Pull request merged**